util/src/test/java/io/kubernetes/client/apimachinery/GroupVersionTest.java

```diff
@@ -21,17 +23,37 @@
 public class GroupVersionTest {
 
   @Test
-  public void parse() {
-    assertEquals(new GroupVersion("", "v1"), GroupVersion.parse(new V1Pod().apiVersion("v1")));
-    assertEquals(
-        new GroupVersion("apps", "v1"),
-        GroupVersion.parse(new V1Deployment().apiVersion("apps/v1")));
-    assertThrows(
-        IllegalArgumentException.class,
-        () -> {
-          GroupVersion.parse(new V1Pod());
-          GroupVersion.parse(new V1Pod().apiVersion(null));
-          GroupVersion.parse(new V1Pod().apiVersion("foo/bar/f"));
-        });
+  public void parseV1() {
+    GroupVersion left = new GroupVersion("", "v1");
+    GroupVersion right = GroupVersion.parse(new V1Pod().apiVersion("v1"));
+    assertThat(left).isEqualTo(right);
+  }
+
+  @Test
+  public void parseAppsV1() {
```

Comment:
7. use `assertj` in tests and proper separate tests in separate methods.

Reply:
Is there value in using assertj? I'd prefer not to add anoother dependency. Can we just use JUnit asserts?